ops/terraform/env/prod/stateful/main.tf

```diff
@@ -16,6 +16,13 @@ module "stateful" {
     allocated_storage = 16000
   }
 
+  db_params = [
+    {name="max_worker_processes", value="16", apply_on_reboot=true},
+    {name="random_page_cost", value="1", apply_on_reboot=false},
+    {name="temp_buffers", value="8192", apply_on_reboot=false},
```

Comment:
Isn't this already the default? https://www.postgresql.org/docs/9.6/runtime-config-resource.html

Reply:
The unit is in chunks of 8k, so the default is 1024 x 8k = 8M.  This bumps it up to 64M.  The postgres doc happily leaves that out, and its very all over the place which values are in which chunks.  Fun times.